client/lib/plans/features-list.js

```diff
@@ -205,11 +205,9 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_EMAIL_SUPPORT_SIGNUP ]: {
 		getSlug: () => constants.FEATURE_EMAIL_SUPPORT_SIGNUP,
-		getTitle: () => i18n.translate( 'Email support' ),
+		getTitle: () => i18n.translate( 'Unlimited email support' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access to unlimited email support.' )` *ES Score: 7*